### DISCUSSION TITLE
Move cursor when scrolling?

### DISCUSSION THREAD

**Discussion by WuerfelDev** [JetBrains/ideavim#704] [Ideas]
In a regular terminal I believe each scroll step is mapped to send the signal of an arrow up key or arrow down key. The cursor is always in the area I'm looking at. It could imagine this as an option for ideavim

**Comment by WuerfelDev** [comment]
Also when holding down j or k it would be cool if that could move multiple lines - not just one

**Reply by citizenmatt** [reply]
Are you on a Mac? This is default behaviour on most operating systems, but not Mac, which has a "press and hold" input method for entering accented characters:

<img width="611" alt="Screenshot 2023-10-26 at 11 03 41 am" src="https://github.com/JetBrains/ideavim/assets/222659/5f14dc7c-421f-4394-b20d-9eb8caffc8e6">

This prevents pressing and holding `j` or `k`, etc. to move up/down the file. This can be disabled, either globally for all apps with:

```
defaults write -g ApplePressAndHoldEnabled -bool false
```

[This StackOverflow page](https://stackoverflow.com/questions/33152551/how-can-i-disable-applepressandholdenabled-for-a-specific-application-repeat) has more details, including how to set it per-application/per-IDE.

**Comment by citizenmatt** [comment]
IdeaVim doesn't currently support this. There is a ticket for it, though, so you can vote and track: [VIM-1948](https://youtrack.jetbrains.com/issue/VIM-1948/Keep-cursor-on-screen-in-Vim-Emulator-when-scrolling-with-mouse-wheel)
